README.md

```diff
@@ -1,84 +1,147 @@
 <a name="readme-top"></a>
 
 <div align="center">
-  <img src="cover-image.png" alt="logo" height="100" width="auto" />
   <br/>
-
-  <h3><b>Module 1 Portfolio</b></h3>
+  <h1><b>Mobile First Portfolio</b></h1>
   <br/>
 </div>
 
 <!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
-- [📗 Table of Contents](#-table-of-contents)
-- [📖 About the project ](#-about-the-project-)
+
+- [� Table of Contents](#-table-of-contents)
+- [📖 Portfolio ](#-portfolio-)
   - [🛠 Built With ](#-built-with-)
     - [Tech Stack ](#tech-stack-)
+  - [⭐️ Key Features ](#️-key-features-)
   - [🚀 Live Demo ](#-live-demo-)
   - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
   - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
   - [🤝 Contributing ](#-contributing-)
   - [⭐️ Show your support ](#️-show-your-support-)
   - [🙏 Acknowledgments ](#-acknowledgments-)
   - [📝 License ](#-license-)
 
 <!-- PROJECT DESCRIPTION -->
-# 📖 About the project <a name="about-project"></a>
+# 📖 Portfolio <a name="about-project"></a>
 
-This is a portfolio site, applying "fisrt mobile" design, workflow in GitHub, pull Request and Linters
-
-**Portfolio** is a web page using "first mobile design" concept. 
+**[Portfolio]** is a site, applying "fisrt mobile" design and RWD (Responsive Web Design).
 
 ## 🛠 Built With <a name="built-with"></a>
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Use HTML and CSS.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Modern Web Browser (Mozilla, Chrome, Edge)</li>
+  </ul>
+</details>
+
+## ⭐️ Key Features <a name="key-features"></a>
+
+- **["First mobile" design]**
+- **[Responsive Web Design]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
+
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> there's no live demo yet.
+- [Live Demo Link] (Cooming soon!)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
+
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> You can follow the Microverse repo's to learn more about this topics
+Open this site in your browser and enjoy our content.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A modern browser for like Mozilla, Chrome, Edge, etc. that supports HTML5 and CSS
+
+### Setup
+
+This application runs directly (it does not need any setup process)
+
+
+### Install
+
+Install this project with:
+
+This application runs directly (it does not need any install process)
+
+
+### Usage
+
+Enter to index.html web page
```

Comment:
- [optional] Kindly take time to look through the edited version of [Microverse readme template](https://github.com/microverseinc/readme-template/edit/master/README.md) and format this sections as applicable to your project. 👍